Paper.tex

```diff
@@ -1795,13 +1795,13 @@ \subsection{Instruction Set}
 &&&& $J_{\text{\tiny JUMPI}}(\boldsymbol{\mu}) \equiv \begin{cases} \boldsymbol{\mu}_\mathbf{s}[0] & \text{if} \quad \boldsymbol{\mu}_\mathbf{s}[1] \neq 0 \\ \boldsymbol{\mu}_{pc} + 1 & \text{otherwise} \end{cases} $ \\
 &&&& This has the effect of writing said value to $\boldsymbol{\mu}_{pc}$. See section \ref{ch:model}. \\
 \midrule
-0x58 & {\small PC} & 0 & 1 & Get the program counter. \\
+0x58 & {\small PC} & 0 & 1 & Get the value of the program counter \textit{before} it's incremented by this instruction. \\
 &&&& $\boldsymbol{\mu}'_\mathbf{s}[0] \equiv \boldsymbol{\mu}_{pc}$ \\
 \midrule
 0x59 & {\small MSIZE} & 0 & 1 & Get the size of active memory in bytes. \\
 &&&& $\boldsymbol{\mu}'_\mathbf{s}[0] \equiv 32\boldsymbol{\mu}_{i}$ \\
 \midrule
-0x5a & {\small GAS} & 0 & 1 & Get the amount of available gas. \\
+0x5a & {\small GAS} & 0 & 1 & Get the amount of gas available after execution of this instruction.  \\
```

Comment:
while literally correct, this has the wrong implication: that gas is reduced during the course of the execution of an operation. gas is reduced, *then* the instruction is executed.